apps/courses/src/routes/privacy-policy/index.tsx

```diff
@@ -0,0 +1,178 @@
+                  variant="spaced"
+                  index={sectionIndexes}
+                  allowMultiple
+                >
+                  {sections.map((section, index) => (
+                    <AccordionItem
+                      id={section.title.replace(/\s+/g, '-').toLowerCase()}
+                      my={8}
+                      onClick={() => toggleAccordionItem(index)}
+                      key={section.title}
+                    >
+                      <Flex alignItems="center">
+                        <CircledNumber
+                          textAlign="center"
+                          border="3px solid"
+                          color="gray.600"
+                          size="2.5rem"
+                          mr="2.5rem"
+                          active={sectionIndexes.includes(index)}
+                          transition=".2s"
+                          onClick={() => toggleAccordionItem(index)}
+                          text={index + 1}
+                        />
+                        <AccordionButton>
+                          <Box flex="1" textAlign="left">
+                            <Heading size="lg" as="h3">
+                              {section.title}
+                            </Heading>
+                          </Box>
+                          <AccordionIcon fontSize="1.5rem" />
+                        </AccordionButton>
+                      </Flex>
+                      <AccordionPanel>{section.content}</AccordionPanel>
+                    </AccordionItem>
+                  ))}
+                </Accordion>
+              </Box>
+            </Box>
+            <Box
+              width={SIDEBAR_WIDTH}
+              flex={`0 0 ${SIDEBAR_WIDTH}px`}
+              display={['none', null, null, 'block']}
+            >
+              <Divider position="fixed" orientation="vertical" />
+              <Box ml={8} position="fixed" width={SIDEBAR_WIDTH}>
+                <List marginTop={4} marginBottom={16} spacing={8}>
+                  {sections.map((section, i) => (
+                    <SectionListItem
+                      key={section.title}
+                      onClick={openAccordionItem}
+                      index={i}
+                      content={section}
+                    />
+                  ))}
+                </List>
+                <Divider my={8} />
+                <Icon
+                  as={FontAwesomeIcon}
+                  icon={faCommentAlt}
+                  size="2x"
+                  my={2}
+                />
+                <Text fontSize="16px">{footer}</Text>
+                <Flex mt={16} mr={4} justify="flex-end">
+                  <Box
+                    cursor="pointer"
+                    onClick={scrollTop}
+                    color="indigo.500"
+                    textAlign="center"
+                  >
+                    <Icon as={FontAwesomeIcon} icon={faChevronUp} />
+                    <Text>TOP</Text>
```

Comment:
Never capitalize your text content. Do this in CSS always otherwise your SEO would look weird where this is typed out. `textTransform="uppercase"`.